### PR TITLE
group vdeletion and alternate views

### PR DESCRIPTION
**Pull request by DeHampster**
Created group deletion page, added different group views for group owner vs non-member, also fixed random bugs

### CODE REVIEW

File: foraging_app/templates/groups/group_delete_confirmation.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <title>Group_Delete_Confirmation</title>
+    <link rel="stylesheet" href="{%  static  'css/delete_group.css' %}">
+</head>
+<body>
+<div class="mainContainer">
+    {% if thisGroup.user_admin.id != request.user.id %}
+        <h1>Wait how'd you get here?</h1>
+        <p>You shouldn't be here.  Please return to the main webpage.</p>
+    {% else %}
+        <h1>Wait!  Are you aure you want to delete {{ thisGroup.name }}</h1>
```
Comment (flickin4) [inline foraging_app/templates/groups/group_delete_confirmation.html]: typo! (aure)

**Review by flickin4 — APPROVED** [review]

File: foraging_app/urls.py [hunk 2]
```diff
@@ -70,6 +71,7 @@
     path('delete/', DeleteUserView.as_view(), name='delete'),
     path('feed/', Feed_View.as_view(), name='feed'),
     path('like/<int:marker_id>/', LikeMarkerView.as_view(), name='like_marker'),
+    path('add_comment_group/<int:marker_id>/<int:groupID>/', AddCommentGroupView.as_view(), name='add_comment_group'),
```
Comment (danzzu) [inline foraging_app/urls.py:74]: Refactor later.

**Review by danzzu — APPROVED** [review]
Looks good!!!

File: foraging_app/templates/groups/group_delete_confirmation.html (hunk 1 above)
Comment (DeHampster) [reply foraging_app/templates/groups/group_delete_confirmation.html]: fixed!

**Review by jjustus19 — APPROVED** [review]

**Pull request merged**